cmd/node_agent/main.go

```diff
@@ -39,14 +39,14 @@ func init() {
 
 	flags := rootCmd.Flags()
 
-	flags.StringVar(&naConfig.ServiceIdentity, "service-identity", "", "Service Identity the node agent is managing")
-	flags.StringVar(&naConfig.ServiceIdentityOrg, "org", "Juju org", "Organization for the cert")
+	flags.StringVar(&naConfig.ServiceIdentity, "service-identity", "default", "Service Identity the node agent is managing")
```

Comment:
Is this a reasonable default? I don't issue any identity called "default".

Reply:
in the beginning I thought it's the right one, updated.